### DISCUSSION TITLE
subscription

### DISCUSSION THREAD

**Discussion by betelopson** [libre-tube/LibreTube#1732] [Q&A] [answered]
How do you import your subs from youtube? I cannot find any info on that. Please help.

**Comment by Bnyro** [comment, marked as answer]
#478, just import the csv file in LibreTube afterwards!

**Comment by betelopson** [comment]
I tried but just keep telling me 'something went wrong'. That is why I'm asking. I tried just the .zip but nothing just the same problem.

**Reply by betelopson** [reply]
I think I need json instead but I don't know if that is a thing to convert it to.

**Reply by Bnyro** [reply]
In theory you just have to import the CSV.
If it doesn't work and you're logged in with a Piped account please try it at e.g. piped.kavin.rocks.
